### DISCUSSION TITLE
[EASTER] Chocolate Bunny

### DISCUSSION THREAD

**Discussion by TheDcol** [sunflower-land/sunflower-land#2339] [Ideas]
An idea for a new Easter themed NFT!

Could be part of the Easter event, where for 2 weeks players will find 'chocolates' from farming crops or collecting eggs, could be a mix of the 2 or just one of those.
Alternatively it could be a good ol' fashioned gas mint war :')

```
**Chocolate Bunny** 

Utility: 
- 1-2% chance of dropping fragments of gold from carrots (random range between 0.1-1 gold)
- tier 1 boost of extra carrots

Costs:
- 1k eggs
- 2k carrots
- 1 easter bunny t1
- (optional) 100 chocolates

Supply: 200-500? More?
```
Yes, the team is focused on land balancing and other stuff so why introduce this silly thing?
Well it could be organised for next year.. and maybe for this year if the team has time.

Very open to feedback, lower boost and more supply? higher costs?

**Comment by future2back** [comment]
1k egg? Impossible...  There needs to be something to do. If not, they're going to buy the resources to mint. We have to make sure there are 1 per account and there are active players.
Fragments of gold should be good at 0.1 lower then 0.35 from Sunflower/Gold Skill.
The possibility of burning lvl 1 rabbits is great... for me 5000  should be good supply.

**Comment by alfa625** [comment]
Assuming previous items is not new player friendly.
You may want to raise the price of the Easter Bunny, but ultimately you want demand to outstrip supply. It is more beneficial to increase demand than to reduce supply. In other words, it is better to organise events that increase the number of players.
If you really want to reward previous players, how about introducing NFTs that are enhanced by combining them, like Cabbage Boy and Cabbage Girl?
Make it so that having both last year's Easter Bunny and this year's Easter NFT will have a better effect than on their own.
I am not sure if this is the right idea as I believe that too much difference between old and new players is not good for the game.

Translated with www.DeepL.com/Translator (free version)
